core/src/test/java/io/questdb/griffin/SqlParserTest.java

```diff
@@ -2388,6 +2399,16 @@ public void testEmptySampleBy() throws Exception {
         assertSyntaxError("select x, y from tab sample by", 30, "literal expected");
     }
 
+    @Test
+    public void testEmptyTableAliasDisallowed() throws Exception {
+        assertSyntaxError("select x from long_sequence(1) ''", 31, "table alias");
+        assertSyntaxError("select x from long_sequence(1) as ''", 34, "table alias");
+        assertSyntaxError("select * from long_sequence(1) \"\"", 31, "table alias");
+        assertSyntaxError("select * from long_sequence(1) as \"\"", 34, "table alias");
```

Comment:
It'd be good to check some combinations of ' and " before and after the dot, e.g.  
'alias.*' 
"alias.*"
'alias'.'*'
"alias"."*"